options_test.go

```diff
@@ -97,7 +97,7 @@ func TestWithLogConsumers(t *testing.T) {
 	// we expect an error because the MySQL environment variables are not set
 	// but this is expected because we just want to test the log consumer
 	require.Error(t, err)
-	require.Contains(t, err.Error(), "container exited with code 1")
+	require.Contains(t, err.Error(), "start container: container exited with code 1")
```

Comment:
```suggestion
	require.Contains(t, err.Error(), "start container: started hook: wait until ready: container exited with code 1")
```

Reply:
This is using `Contains` so it only has to check for base error we're expecting. This specifically avoids checking any wrapping that might happen as that may change making the test fragile, is as close to `errors.Is` as we can get.